kustomize/overlays/ci/shard/kustomization.yaml

```diff
@@ -1,17 +1,9 @@
-secretGenerator:
-- name: event-bridge-shard-operator-secrets
```

Comment:
@vaibhavjainwiz I think this is defined like this to allow the CI pipeline to replace values from GitHub Secrets. @sutaakar recently migrated the CI pipeline to use sso.redhat.com for auth.

So your choices are:

1. Keep things in the `kustomization.yaml` as the `literal` secret definition, but change the `name` of the `secret` to be `addon-smart-events-operator-parameters`.
2. Update https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/.github/workflows/Manager-CI.yaml#L100-L114 to work with your new `.yaml` file

In either case, can you please update the values of keys that are replaced by the CI pipeline to something like `TBD`. It's quite confusing that we have values here that are not actually used e.g:

From 

```
- WEBHOOK_CLIENT_SECRET=GMJiM0XNkp7b9bK0wP47odAtKxc4ufRv
```
to

```
- WEBHOOK_CLIENT_SECRET=REPLACE_ME
```

@sutaakar Can you double-check everything I'm saying is correct.

cheers,
Rob

Reply:
I modify my PR to adopt **approach 2**.

Reply:
@robpblake right